artifactory/commands/golang/go.go

```diff
@@ -67,18 +78,65 @@ func (gc *GoCommand) SetGoArg(goArg []string) *GoCommand {
 	return gc
 }
 
+func (gc *GoCommand) CommandName() string {
+	return GoCommandName
+}
+
 func (gc *GoCommand) ServerDetails() (*config.ServerDetails, error) {
+	// If deployer Artifactory details exists, returs it.
 	if gc.deployerParams != nil && !gc.deployerParams.IsServerDetailsEmpty() {
 		return gc.deployerParams.ServerDetails()
 	}
-	return gc.resolverParams.ServerDetails()
-}
 
-func (gc *GoCommand) CommandName() string {
-	return GoCommandName
+	// If resolver Artifactory details exists, returs it.
+	if gc.resolverParams != nil && !gc.resolverParams.IsServerDetailsEmpty() {
+		return gc.resolverParams.ServerDetails()
+	}
+
+	// If conf file exists, return the server configured in the conf file.
+	if gc.configFilePath != "" {
```

Comment:
I that this this file always exists now, right? The code still assumes that it may not exist. Let's compare this behaviour with other commands.